Makefile.custom.mk

```diff
@@ -102,8 +102,9 @@ test-integration: ginkgo ensure-gcp-envs ## Run integration tests
 	rm $(GOOGLE_APPLICATION_CREDENTIALS)
 
 .PHONY: test-acceptance
+test-acceptance: KUBECONFIG=$(HOME)/.kube/$(CLUSTER).yml
 test-acceptance: ginkgo ensure-gcp-envs deploy-acceptance-cluster ## Run acceptance testst
-	KUBECONFIG="$(HOME)/.kube/$(CLUSTER).yml" $(GINKGO) -p -nodes 2 -r -randomize-all --randomize-suites tests/acceptance
+	KUBECONFIG="$(KUBECONFIG)" $(GINKGO) -p -nodes 2 -r -randomize-all --randomize-suites tests/acceptance
```

Comment:
isn't `KUBECONFIG="$(KUBECONFIG)"` kind of superfluous?

Reply:
Actually that's half true I think. According to [docs](https://www.gnu.org/software/make/manual/html_node/Environment.html):

> When make runs a recipe, variables defined in the makefile are placed into the environment of each shell. This allows you to pass values to sub-make invocations (see [Recursive Use of make](https://www.gnu.org/software/make/manual/html_node/Recursion.html#Recursion)). By default, only variables that came from the environment or the command line are passed to recursive invocations. You can use the export directive to pass other variables. See [Communicating Variables to a Sub-make](https://www.gnu.org/software/make/manual/html_node/Variables_002fRecursion.html#Variables_002fRecursion), for full details.

So if you had KUBECONFIG set beforehand it would work, but if you don't it won't unless line 105 uses `export`